test/models/journeyDomain/transport/TransportMeansDomainSpec.scala

```diff
@@ -19,35 +19,40 @@ package models.journeyDomain.transport
 import base.SpecBase
 import generators.Generators
 import models.InlandMode
+import models.InlandMode._
 import models.domain.{EitherType, UserAnswersReader}
-import InlandMode._
+import models.reference.Nationality
+import org.scalacheck.Arbitrary.arbitrary
 import org.scalacheck.Gen
 import pages.QuestionPage
-import pages.transport.transportMeans.departure.InlandModePage
+import pages.transport.transportMeans.departure.{InlandModePage, VehicleCountryPage}
 
 class TransportMeansDomainSpec extends SpecBase with Generators {
 
-  val inlandModeList: Seq[InlandMode] = Seq(
-    Maritime,
-    Rail,
-    Road,
-    Air,
-    Mail,
-    Fixed,
-    Waterway,
-    Unknown
-  )
+  "TransportMeansDomain" - {
 
-  val inlandMode: InlandMode = Gen.oneOf(inlandModeList).sample.value
+    val inlandModeList: Seq[InlandMode] = Seq(
+      Maritime,
+      Rail,
+      Road,
+      Air,
+      Mail,
+      Fixed,
+      Waterway,
+      Unknown
+    )
 
-  "TransportMeansDomain" - {
+    val inlandMode: InlandMode   = Gen.oneOf(inlandModeList).sample.value
```

Comment:
Looks like we have an arbitrary inland mode defined in ModelGenerators. So all of this can just become
`val inlandMode: InlandMode = arbitrary[InlandMode].sample.value`